### DISCUSSION TITLE
Can I ask what is this {sb_order_seq} at the end url for Orders end point

### DISCUSSION THREAD

**Discussion by oarnb** [Sellbrite/Sellbrite-API#14] [Q&A]
Can I ask what is this {sb_order_seq} at the end url for Orders end point

![Screenshot_6](https://github.com/Sellbrite/Sellbrite-API/assets/138830828/2e5322b0-2b7b-4b77-b0a8-b76f803b4bb0)

**Comment by Taegost** [comment]
That correlates to the Order # in the GUI. I actively use it in my automation that pulls all open orders from SellBrite to filter out the orders have already processed.

![image](https://github.com/Sellbrite/Sellbrite-API/assets/22455168/491761ae-1a6e-48fb-86c6-05899da6bb4a)

**Reply by oarnb** [reply]
Will the URL be like this when I used it in the HTTP module? See attached

![Screenshot_10](https://github.com/Sellbrite/Sellbrite-API/assets/138830828/dfb4163b-4319-4c36-8063-14ae24aafe20)

**Reply by Taegost** [reply]
Sorry I didn't respond sooner, I check these discussions about once a week and for some reason this never showed up as having an unread message until today. Yes, that would be correct, just make sure to use the actual order number without any special characters (like the : in your example, ie: _https://api.sellbrite.com/v1/orders/1001_).
